3ds/libsidplayfp/PKGBUILD

```diff
@@ -0,0 +1,30 @@
+# Maintainer:  Dave Murphy <davem@devkitpro.org>
+# Contributor: Sebastian Weber <me@badda.de>
+
+pkgname=3ds-libsidplayfp
+pkgver=1.8.8
+pkgrel=1
+pkgdesc="Library to play Commodore 64 music files (SID-files) derived from libsidplay2"
+arch=('any')
+url="https://sourceforge.net/projects/sidplay-residfp/"
+license=('GPLv2')
+options=(!strip libtool staticlibs)
+source=(
+	"https://netcologne.dl.sourceforge.net/project/sidplay-residfp/libsidplayfp/1.8/libsidplayfp-1.8.8.tar.gz"
```

Comment:
This is a specific mirror. Replace the url here with `https://netcologne.dl.sourceforge.net/project/sidplay-residfp/libsidplayfp/${_basever}/libsidplayfp-${pkgver}.tar.gz`

Reply:
Just out of curiosity, what was the error?

Reply:
I just tried again - the error is not occurring anymore, seems like sourceforge had only temporary issues:
curl: (35) OpenSSL SSL_connect: SSL_ERROR_SYSCALL in connection to downloads.sourceforge.net:443

Reply:
It's just SourceForge mirror system messing up. It sometimes times out or returns a mirror that's currently down.